sdk/core/azure-core/azure/core/pipeline/policies/retry.py

```diff
@@ -101,8 +163,10 @@ def configure_retries(self, options):
         }
 
     def get_backoff_time(self, settings):
-        """ Formula for computing the current backoff
```

Comment:
"Formula" read very strange to me here. Maybe "Return the current computed backoff" or "Compute the current backoff" ?

Reply:
Agree with Bryan. It is just a very simple logic that return minimal (max, backoff_value). We can just put as simple as "Return backoff time"